components/Sidebar/ChatHistory.tsx

```
@@ -47,6 +49,8 @@ export default function ChatHistory({
   const pathname = usePathname();
   const [chatToDelete, setChatToDelete] = React.useState<Chat | null>(null);
   const [sessionId, setSessionId] = React.useState<string | null>(null);
+  const { navigate } = useNavigation();
+  const { isMobile, setOpenMobile } = useSidebar();
```

ESLint

[error] 'isMobile' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)